tasks/pg_utils/database.py

```diff
@@ -113,14 +155,21 @@ def get_cursor(dbconnect_info):
             cursor.close()
 
 
-
-def single_query(sql_stmt, dbconnect_info, params=None):
+def single_query(sql_stmt, dbconnect_info: dict[str, Union[str, int]], params=None):
```

Comment:
Should there be a -> <type> in this function definition?

Reply:
There should be, but I'm a bit unsure of the type. Added what I could.